hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -157,7 +157,7 @@ if {[info exists ::env(CHECK)]} {
 # propagate to "relevant signals" (by default, top instance outputs). If you need to specify
 # include/exclude relevant signals manually, run "jg_auto_coi_cov_waivers -help" for more
 # options.
-jg_auto_coi_cov_waivers
+# jg_auto_coi_cov_waivers
```

Comment:
perhaps add a TODO, if we should enable this later.

Reply:
sounds good, will update that! Thanks